### DISCUSSION TITLE
Linkage Checker error

### DISCUSSION THREAD

**Discussion by apfelnymous** [INSPIRE-MIF/helpdesk-validator#866] [Q&A] [answered]
Using the linkage checker, it would always return "The request had an error! Please try again later or contact the support"

Anyone knows more ?

**Comment by fabiovinci** [comment]
Dear @apfelnymous,

the "Resource linkages checker tool" is managed by the Geoportal team, you should open an issue in the [related helpdesk](https://github.com/INSPIRE-MIF/helpdesk-geoportal/issues).

**Comment by apfelnymous** [comment, marked as answer]
This was solved as wrong links were used
